HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -1,19 +1,32 @@
 import FWCore.ParameterSet.Config as cms
 
-# CMSSW version specific customizations
-import os
-cmsswVersion = os.environ['CMSSW_VERSION']
-
-if cmsswVersion >= "CMSSW_7_5":
 
 # Simplified TrackerTopologyEP config (PR #7966)
-    if 'trackerTopology' in locals():
-        paras = tuple(vars(trackerTopology)['_Parameterizable__parameterNames'])
-        for para in paras:
-          delattr(trackerTopology,para)
-        setattr(trackerTopology,'appendToDataLabel',cms.string(""))
```

Comment:
This is how I foresee to keep this: one customisation function per PR, and a top-level `customise()` to call all the needed ones.